dist/index.js

```diff
@@ -5885,9 +5886,13 @@ async function main() {
                 branch,
             });
             console.log(`Found ${data.total_count} runs total.`);
-            const runningWorkflows = data.workflow_runs.filter(run => run.head_branch === branch && run.head_sha !== headSha && run.status !== 'completed' &&
+            console.log(data.workflow_runs.map(run => `- ${run.html_url}`).join('\n'));
```

Comment:
From my tests, `data.workflow_runs` seems to return runs from other branches too; which can leads to a really big list.

This is probably why you do `run => run.head_branch === branch` for `runningWorkflows`.

What would you think if I do something like this instead?
```ts
console.log(`Found ${data.total_count} runs total for branch ${branch}.`);
console.log(data.workflow_runs.filter(run => run.head_branch === branch).map(run => `- ${run.html_url}`).join('\n'));
```

Reply:
Let's revert the console.log() changes since these are not necessary for this PR

Reply:
Good idea. I've made a separated PR https://github.com/styfle/cancel-workflow-action/pull/51 if this is something that would interest you.